installimg/8.2.updates/etc/systemd/system/getty@tty2.service

```diff
@@ -0,0 +1,41 @@
+#  This file is based on the service file from systemd.
```

Comment:
This service has literally nothing to do with the installer, it is really rather part of the "installimg distro", just to have a getty running.  E.g. if we were to replace `host-installer` by another program, it would still fill its role.